.circleci/config.yml

```diff
@@ -61,7 +61,7 @@ references:
     keys:
       - v{{ .Environment.GLOBAL_CACHE_PREFIX }}-v8-jest-{{ .Environment.CIRCLE_JOB }}-{{ .Environment.CIRCLE_NODE_INDEX }}/{{ .Environment.CIRCLE_NODE_TOTAL }}-{{ .Branch }}-{{ .Revision }}
       - v{{ .Environment.GLOBAL_CACHE_PREFIX }}-v8-jest-{{ .Environment.CIRCLE_JOB }}-{{ .Environment.CIRCLE_NODE_INDEX }}/{{ .Environment.CIRCLE_NODE_TOTAL }}-{{ .Branch }}
-      - v{{ .Environment.GLOBAL_CACHE_PREFIX }}-v8-jest-{{ .Environment.CIRCLE_JOB }}-{{ .Environment.CIRCLE_NODE_INDEX }}/{{ .Environment.CIRCLE_NODE_TOTAL }}-master
+      - v{{ .Environment.GLOBAL_CACHE_PREFIX }}-v8-jest-{{ .Environment.CIRCLE_JOB }}-{{ .Environment.CIRCLE_NODE_INDEX }}/{{ .Environment.CIRCLE_NODE_TOTAL }}-HEAD
```

Comment:
I don't think this will match, it corresponds to some special variables listed here:

https://circleci.com/docs/2.0/caching/#using-keys-and-templates

It's not a big deal if we temporarily have some cache misses here. I think this will need to be updated whenever the default branch is changed.

Reply:
> There are a bunch of branch filters in the workflows section that will need to be updated as well.

I noticed these but wasn't sure that updating them to `HEAD` would work given the "branch name" we're merging into currently is still the old one. My plan was to offset those changes for when we're actually changing the branch name (once GitHub makes the change we're hoping there will be some tooling to help out that is new-main-branch-name agnostic)

> I think this will need to be updated whenever the default branch is changed.

If we leave these at HEAD then it should be fine, right? Once the cache misses resolve it'll be okay?

Reply:
Alternatively, if these don't actually reference the branch, then I can go ahead and change it to `trunk`

Reply:
They are intended to reference the default branch, but if they miss for a period of time, it shouldn't be a big deal. It seems fine to update them 👍